cpp/oneapi/dal/algo/kmeans/backend/gpu/infer_kernel_dpc.cpp

```diff
@@ -40,10 +40,11 @@ using daal_kmeans_lloyd_dense_ucapi_kernel_t =
     daal_kmeans::internal::KMeansDenseLloydBatchKernelUCAPI<Float>;
 
 template <typename Float>
-struct infer_kernel_gpu<Float, method::by_default> {
-    infer_result operator()(const dal::backend::context_gpu& ctx,
-                            const descriptor_base& params,
-                            const infer_input& input) const {
+struct infer_kernel_gpu<Float, method::by_default, task::clustering> {
+    infer_result<task::clustering> operator()(const dal::backend::context_gpu& ctx,
+                                              const descriptor_base<task::clustering>& params,
+                                              const infer_input<task::clustering>& input) const {
+        using Task = task::clustering;
```

Comment:
This alias is just for a single occurrence. Maybe it ins't necessary?

Reply:
fixed